src/components/shared/HeaderMessage.tsx

```diff
@@ -0,0 +1,13 @@
+interface props {
+  heading: string;
+  subheading: string;
+}
+
+export default function HeaderMessage({ heading, subheading }: props) {
+  return (
+    <div className="inline-flex md:flex lg:flex flex-col items-start gap-2">
```

Comment:
```suggestion
    <div className="flex flex-col items-start gap-2">
```
No need to use `inline-flex` here. You also don't need to specify `flex` for every single breakpoint because mobile, tablet, and desktop all use `flex`. Start by doing mobile first. Then, expand your screen size to see if there should be any changes for tablet and then desktop. By specifying just `flex`, it will apply that class to every single breakpoint.